Content.Client/_DV/Overlays/GlimmerOverlaySystem.cs

```diff
@@ -0,0 +1,61 @@
+using Content.Shared._DV.CCVars;
+using Content.Shared.Psionics.Glimmer;
+using Robust.Client.Graphics;
+using Robust.Shared.Configuration;
+
+namespace Content.Client._DV.Overlays;
+
+public sealed partial class GlimmerOverlaySystem : EntitySystem
+{
+    [Dependency] private readonly IOverlayManager _overlayMan = default!;
+    [Dependency] private readonly IConfigurationManager _cfg = default!;
+
+    private GlimmerOverlay _overlay = default!;
+
+    public override void Initialize()
+    {
+        base.Initialize();
+
+        _overlay = new GlimmerOverlay();
+        SubscribeNetworkEvent<GlimmerChangedEvent>(OnGlimmerChanged);
+        Subs.CVar(_cfg, DCCVars.DisableGlimmerShader, OnDisableGlimmerShaderChanged);
+    }
+
+    private void OnGlimmerChanged(GlimmerChangedEvent eventArgs)
+    {
+        if(_cfg.GetCVar(DCCVars.DisableGlimmerShader))
```

Comment:
cache the cvar and update the cache when cvar changes

like Subs.CVar but by hand with OnValueChanged or whatever